src/Action/AbstractPostArchiveAction.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\NewsBundle\Action;
+
+use Sonata\NewsBundle\Model\BlogInterface;
+use Sonata\NewsBundle\Model\PostManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+abstract class AbstractPostArchiveAction extends Controller
+{
+    /**
+     * @var BlogInterface
+     */
+    private $blog;
+
+    /**
+     * @var PostManagerInterface
+     */
+    private $postManager;
+
+    /**
+     * AbstractPostArchiveAction constructor.
+     *
+     * @param BlogInterface        $blog
+     * @param PostManagerInterface $postManager
```

Comment:
This can be removed too IMO, and I thought you were against removing it but now I see you did not put it on some methods below.